app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+from typing import List
+
+from app.models import Actor
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect("/Users/khrypunov/"
+                                          "PycharmProjects/"
+                                          "py-actor-manager/cinema.sqlite")
```

Comment:
The database connection path is hardcoded. Consider using a configuration file or environment variables to make the database path more flexible and enhance portability.